mir-module/src/main/resources/META-INF/resources/js/epusta.js

```diff
@@ -55,40 +55,40 @@ OASInline.prototype= {
         this.$element.text("");
     }
   }
-  
+
   ,setCount: function (count) {
     this.count=count;
   }
-  
+
   ,getCounttype: function (counttype) {
     return(this.counttype);
   }
 };
 
-OASInline.receiveData = function(oasinline,xml) {
-  
+ePuStaInline.receiveData = function(epustainline,xml) {
+
   if (xml) {
     var nodes = $(xml).find("access");
     nodes.each (function () {
       var type=$($(this).children( "type" )[0]).text();
-      if (type==oasinline.getCounttype()) {
+      if (type==epustainline.getCounttype()) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected '===' and instead saw '=='.](https://app.codacy.com/manual/MyCoRe/mir/pullRequest?prid=5639065)